src/main/java/org/tikv/raw/RawKVClient.java

```diff
@@ -36,16 +54,29 @@ public static RawKVClient create(String address) {
     return new RawKVClient(address);
   }
 
+  @Override
+  public void close() {
+    session.close();
+  }
+
   /**
    * Put a raw key-value pair to TiKV
    *
    * @param key raw key
    * @param value raw value
    */
   public void put(ByteString key, ByteString value) {
-    Pair<TiRegion, Metapb.Store> pair = regionManager.getRegionStorePairByKey(key);
-    RegionStoreClient client = RegionStoreClient.create(pair.first, pair.second, session);
-    client.rawPut(defaultBackOff(), key, value);
+    BackOffer backOffer = defaultBackOff();
+    while (true) {
+      Pair<TiRegion, Metapb.Store> pair = regionManager.getRegionStorePairByKey(key);
+      RegionStoreClient client = RegionStoreClient.create(pair.first, pair.second, session);
```

Comment:
For a lightweight action like put, reconstruct client each time might cost too much on itself and GC. Likely we put a todo here and cache client later if we have time.